src/types/storybook.d.ts

```diff
@@ -1,7 +1,10 @@
 import { Story } from '@storybook/react/types-6-0'
+import { AnyFunction } from 'types'
+
+type Bind = AnyFunction['bind']
 declare module '@storybook/addons' {
-  export interface BaseStory<Args, StoryFnReturnType> {
-    bind(thisArg: Parameters<Function['bind']>[0]): Story<Args>
-    bind(...args: Parameters<Function['bind']>): ReturnType<Function['bind']>
+  export interface BaseStory<Args> {
+    bind(thisArg: Parameters<Bind>[0]): Story<Args>
```

Comment:
I don't understand this change

Reply:
what do you mean? New rules forbid (or strongly discourage) using `Function, {}, etc.` that's why replaced with `AnyFunction` so as not to populate code with `(...args: unknown[]) => unknown`